### ISSUE TITLE
Mocha as generated dev dependency

### ISSUE THREAD

**Issue by betaorbust** [eslint/generator-eslint#51]
Running through a fresh install, the generated files don't have `mocha` as a dev dependency, but the supplied test command relies on it. 

Repro steps:
- Set up a new plugin: `yo eslint:plugin`
- Set up a new rule: `yo eslint:rule`
- Install dependencies: `npm install`
- Try to run test `npm test`
- Test run fails with following message:  
```
> mocha
sh: mocha: command not found
npm ERR! Test failed.  See above for more details.
```

Proposed Resolution:
Add `mocha` as a dev dependency for the generated project.

**Comment by platinumazure** [comment]
Agreed, this is a bug. Please feel free to submit a PR if you like. Thanks for the report!

**Comment by platinumazure** [comment]
@betaorbust Are you using latest `generator-eslint`? As far as I can tell, the generated package.json file does specify mocha as a devDependency.

**Comment by betaorbust** [comment]
My apologies. Had 1.0.2 installed. This was taken care of in https://github.com/eslint/generator-eslint/commit/2a37eb5d36644d3d9855c696120fdae73b272192 and published in 1.1.1.
![](https://img.pandawhale.com/post-10040-I-do-not-like-the-cone-of-sham-XbwM.gif)
